tests/test_models/test_vid/test_dff.py

```diff
@@ -76,9 +76,10 @@ def test_dff_forward_predict_mode(self, cfg_file, devices):
                 for i in range(3):
                     packed_inputs = demo_mm_inputs(
                         batch_size=1, frame_id=i, num_ref_imgs=0)
-                    batch_inputs, data_samples = model.data_preprocessor(
-                        packed_inputs, False)
+                    out_data = model.data_preprocessor(packed_inputs, True)
```

Comment:
```suggestion
                    out_data = model.data_preprocessor(packed_inputs, False)
```
please change it for all unittest of model